extensions/schoology/.gitignore

```diff
@@ -1,10 +0,0 @@
-# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
-
-# dependencies
-/node_modules
-
-# Raycast specific files
-raycast-env.d.ts
-
-# misc
-.DS_Store
```

Comment:
Why did you delete this file? Sounds like a keeper to me 🙂

Reply:
Idk. I deleted it because github was giving me issues.